src/opnsense/www/js/opnsense_ui.js

```diff
@@ -486,6 +486,32 @@ function initFormAdvancedUI() {
     });
 }
 
+/**
+ * handle keyboard shortcuts for toggling advanced and help
+ */
+function initGlobalOpenShortcuts() {
+    $(document).off('keydown.opnsenseGlobalOpenToggles').on('keydown.opnsenseGlobalOpenToggles', function (e) {
+        if (e.ctrlKey || e.altKey || e.metaKey) return;
+        const t = e.target;
+        const tag = (t.tagName || '').toLowerCase();
+        if (tag === 'input' || tag === 'textarea' || tag === 'select' || t.isContentEditable) return;
+
+        if (e.key === 'a' || e.key === 'A') {
+            const $adv = $('[id*="show_advanced"]').first();
+            if ($adv.length) {
+                $adv.click();
+                e.preventDefault();
+            }
+        } else if (e.key === 'h' || e.key === 'H') {
+            const $help = $('[id*="show_all_help"]').first();
```

Comment:
I wonder if this will open the full help of the main form when a modal is open on top of it (when the user meant to open the full help of the modal). Case in point: unbound -> query forwarding -> open dialog for a new entry, press `h`

Reply:
At the current form, the answer is both. That's kind of a unique scenario actually. Give me some time to try and cook something up.